ang/civicase/ActivityActions--tags.html

```diff
@@ -0,0 +1,24 @@
+<div id="bootstrap-theme" class="civicase__tags-modal">
+  <form class="form-horizontal">
+    <div class="form-group civicase__tags-modal__generic-tags__container">
+      <div ng-repeat="tag in model.genericTags">
+        <input
+          id="civicase_tags_{{tag.id}}"
+          type="checkbox" value="{{tag.name}}"
+          ng-click="model.toggleGenericTags(model, tag.id)"/>
+        <label for="civicase_tags_{{tag.id}}">{{'&nbsp;'.repeat(tag.indentationLevel)}}{{tag.name}}</label>
```

Comment:
Adding `&nbsps` will restrict the width for a tab as width of non-breaking space can't be increased specifically. Do you think we should use empty `span` tags and add `padding-left/margin-left` to them will work better?

If we are sure that we need single space character width for one indentation then we can go with the current implementation, but if we want to add indentation char width through css then adding html element will be better. 

This is just a recommendation. Please feel free to provide your inputs on this.

Reply:
I agree, I am using `margin-left` now, thanks.